Makefile

```diff
@@ -697,6 +697,10 @@ ifeq ($(UNAME), Darwin)
 	install_name_tool -id $(CURDIR)/$(BIN_DIR)/libHalide.$(SHARED_EXT) $(BIN_DIR)/libHalide.$(SHARED_EXT)
 endif
 
+zynq_runtime: $(LIB_DIR)/libHalide.a $(ROOT_DIR)/apps/hls_examples/hls_support/HalideRuntimeZynq.cpp $(INCLUDE_DIR)/HalideRuntime.h
```

Comment:
Do you need `$(LIB_DIR)/libHalide.a` in the dependency?

Reply:
Good catch. Actually it's not necessary. I will remove that.